features/Context/ContentEdit.php

```diff
@@ -0,0 +1,63 @@
+<?php
+/**
+ * @license For full copyright and license information view LICENSE file distributed with this source code.
+ */
+namespace EzSystems\RepositoryForms\Features\Context;
+
+use Behat\Behat\Context\Context;
+use Behat\Behat\Context\SnippetAcceptingContext;
+use Behat\MinkExtension\Context\MinkContext;
+use PHPUnit_Framework_Assert as Assertion;
+
+class ContentEdit extends MinkContext implements Context, SnippetAcceptingContext
+{
+    /**
+     * Name of the content that was created using the edit form. Used to validate that the content was created.
+     * @var string
+     */
+    private $createdContentName;
+
+    /**
+     * @Then /^I should see a folder content edit form$/
```

Comment:
Why use this notation? Why not just `@Then I should see a folder content edit form` ?